src/App.tsx

```diff
@@ -55,6 +57,35 @@ loader.init().then((monaco) => {
   })
 })
 
+const getAppDataFromSearchParams = (): AppData => {
+  const urlParams = new URLSearchParams(window.location.search)
+  const compressedAppData = urlParams.get('appData')
+  if (!compressedAppData)
+    return {
+      schema: '',
+      values: [],
+    }
+
+  const decompressedAppData =
+    LZString.decompressFromEncodedURIComponent(compressedAppData)
+  const AppData = JSON.parse(decompressedAppData)
```

Comment:
Variable name should begin with lower case.
Also, the previous declaration can stay on a single line, maybe prettier config is too strict.
Lastly, would leave an empty line below before the return statement.